keylime/cloud_verifier_common.py

```diff
@@ -142,7 +142,7 @@ def init_mtls(section='cloud_verifier', generatedir='cv_ca'):
         tls_dir = os.path.abspath('%s/%s' % (common.WORK_DIR, 'cv_ca'))
         if not os.path.exists("%s/cacert.crt" % (tls_dir)):
             raise Exception(
-                "It appears that the verifier has not yet created a CA and certificates, please run the verifier first")
+                "It appears that the verifier has not yet created a CA and tpm_cert_store, please run the verifier first")
```

Comment:
This is actually a different certificate store `ca`  to the `tpm_cert_store`. The ca is where new certs are created for communication between the systems, where as the `tpm_cert_store` is where we keep vendor tpm public certificates.

Reply:
This file made some automatic changes when I renamed the folder from ’certificates’ to ’tpm_cert_store’. I can change it back to it’s initial state manually :).